module/enrichers.mjs

```diff
@@ -967,11 +1045,45 @@ async function rollAction(event) {
     try {
       switch ( type ) {
         case "damage": return await rollDamage(event);
+        case "item":
+          let item;
+	          // If UUID is provided, always roll that item directly
+	          if ( target.dataset.rollItemUuid ) item = await fromUuid(target.dataset.rollItemUuid);
+	          else if ( target.dataset.rollItemName ) {
```

Comment:
Could we split this branch out into its own function, maybe `findItem` or something?